main.py

```diff
@@ -0,0 +1,26 @@
+from models.fish import Fish
+from models.bird import Bird
+from models.mammal import Mammal
+from models.insect import Insect
+from models.reptile import Reptile
```

Comment:
Not a problem, but just another way to do imports here: create an ```__init__.py``` file inside of your models folder, import all your classes inside of ```__init__.py```, and then you'll be able to simply do something like this here:
```
from models import (
    Fish,
    Bird,
    Mammal,
    Insect,
    Reptile,
    ...
)
```
Which is a more pythonic way to deal with imports when you have your classes organized in folders. But still, it's NOT a mistake to just import from files.
Useful link: [Understanding python imports](https://towardsdatascience.com/understanding-python-imports-init-py-and-pythonpath-once-and-for-all-4c5249ab6355)